### ISSUE TITLE
Error: Resource not accessible by integration

### ISSUE THREAD

**Issue by kevnm67** [jusx/mergeable#118]
Recently, Mergeable started reporting the error "Resource not accessible by integration".  Not sure what's causing this.  Any ideas?

<img width="678" alt="screen shot 2018-12-19 at 8 50 04 am" src="https://user-images.githubusercontent.com/2734154/50224266-70c73c00-036b-11e9-968b-76a38f069c14.png">

**Comment by jusx** [comment]
Hi @kevnm67 

Are you using the `owners` option by any chance?

**Comment by shine2lay** [comment]
@jusx i think this will be fixed when we add permission to access the `Owners` file path

**Comment by jusx** [comment]
@kevnm67 the `owner` property requires the Mergeable to have Read Only access to the contents of the repository. While GitHub does have a permission setting that allows us to specify a file that the app can access like `.github/mergeable.yml` there isn't a way for us to set permissions for a set of files.

GitHub allows an app to have RO for one file or entire contents of a private repos. 

Because of this the `owners` property works for public repositories but not so for private because of this. We have been staying away from requesting RO for entire contents of the repo but that also prevents us from automating more for private repos.  

We will be updating the GitHub App settings to request for RO permission of private repos contents. You will receive an email asking you to approve this request -- once you approve this request, mergeable should work properly with the `owners` option within private repos.

**Comment by kevnm67** [comment]
@jusx Nope.  I’m using the main app. I removed the owners setting.  For now I just added our names to the reviewers.  

I’ve considered running my own instance.  When I tried running it locally I had some issues.  It was a few weeks back and not sure why.  I’ll open a ticket if I can’t get it working when I revisit it.
